ewm-service/src/main/java/ru/practicum/ewmservice/category/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,32 @@
+package ru.practicum.ewmservice.category.mapper;
+
+import org.springframework.stereotype.Component;
+import ru.practicum.ewmservice.category.dto.CategoryDto;
+import ru.practicum.ewmservice.category.dto.NewCategoryDto;
+import ru.practicum.ewmservice.category.entity.CategoryEntity;
+
+@Component
+public class CategoryMapper {
+    public static CategoryDto toCategoryDto(CategoryEntity entity) {
+        CategoryDto dto = CategoryDto.builder()
+                .id(entity.getId())
+                .name(entity.getName())
+                .build();
+        return dto;
```

Comment:
Лучше не создавать лишние переменные, можно сразу возвращать результат бидера:
```suggestion
        return CategoryDto.builder()
                .id(entity.getId())
                .name(entity.getName())
                .build();;
```

Поправь в методах ниже, пожалуйста